src/main/scala/mesosphere/marathon/io/IO.scala

```diff
@@ -75,5 +75,29 @@ object IO extends StrictLogging {
         0
     }
   }
+
+  def transfer(from: InputStream, to: OutputStream): Long = transfer(Some(from), Some(to))
+
+  // TODO(karsten): We might want to use the TarFlow
```

Comment:
So  far the tar flow is too specific and I wanted to get everything working first.